doc/reference/autoinstall-reference.rst

```diff
@@ -432,52 +435,102 @@ In the new format, the ``primary`` section expects a list of mirrors, which can
+          - uri: "$CUSTOM_MIRROR_2"
+
+  # Use one mirror for amd64 and another for i386.
+  autoinstall:
+    apt:
+      mirror-selection:
+        primary:
+          - uri: "http://jp.archive.ubuntu.com/ubuntu"
+            arches: [amd64]
+          - uri: "http://tw.archive.ubuntu.com/ubuntu"
+            arches: [i386]
 
 fallback
 ^^^^^^^^
 
 * **type:** string (enumeration)
-* **default:** offline-install
+* **default:** ``offline-install``
 
 Controls what Subiquity does when no primary mirror is usable. Supported values are:
 
 * ``abort``: abort the installation
 * ``offline-install``: revert to an offline installation
 * ``continue-anyway``: attempt to install the system anyway (not recommended; the installation fails)
 
+Examples:
+
+.. code-block:: yaml
+
+    # Only install from the primary archive and abort the installation if mirror validation fails.
+    autoinstall:
+      apt:
+        mirror-selection:
+          primary:
+            - uri: "http://archive.ubuntu.com/ubuntu"
+        fallback: abort
+
+    # Only install from the German country mirror and continue with an offline install if mirror validation fails.
+    autoinstall:
+      apt:
+        mirror-selection:
+          primary:
+            - uri: "http://de.archive.ubuntu.com/ubuntu"
+        fallback: offline-install
+
 geoip
 ^^^^^
 
 * **type:** boolean
 * **default:** ``true``
 
-If ``geoip`` is set to ``true`` and one of the candidate primary mirrors has the special value ``country-mirror``, a request is made to ``https://geoip.ubuntu.com/lookup``. Subiquity then sets the mirror URI to ``http://CC.archive.ubuntu.com/ubuntu`` (or similar for ports) where ``CC`` is the country code returned by the lookup. If this section is not interactive, the request expires after 10 seconds.
+If ``geoip`` is set to ``true`` and one of the candidate primary mirrors has the special value ``country-mirror``, a request is made to ``https://geoip.ubuntu.com/lookup``. Subiquity then sets the mirror URI to ``http://CC.archive.ubuntu.com/ubuntu`` where ``CC`` is the country code returned by the lookup. If this section is not interactive, the request expires after 10 seconds.
 
 If the legacy behaviour (i.e., without mirror-selection) is in use, the geolocation request is made if the mirror to be used is the default, and its URI is replaced by the proper country mirror URI.
 
-To specify a mirror, use a configuration like this:
+Examples:
 
 .. code-block:: yaml
 
+    # Use the automatically determined country mirror first, followed by an explicit backup mirror.
     autoinstall:
       apt:
         mirror-selection:
           primary:
-            - uri: YOUR_MIRROR_GOES_HERE
             - country-mirror
-            - uri: http://archive.ubuntu.com/ubuntu
-
-To add a PPA:
+            - uri: http://dk.archive.ubuntu.com/ubuntu
+        geoip: true
 
-.. code-block:: yaml
+    # Don't use the country mirror and immediately fallback to next listed mirror.
```

Comment:
This example is a bit confusing IMO :upside_down_face: 

It would make sense if the user could specify `country: <cc>` but as-is specifying `country-mirror` and `geoip: false` feels wrong.

Reply:
Yes I was conflicted about this. I think the ability to set `geoip: false` with the new mirror selection format feels wrong/pointless but I wanted to document that it technically is just a skip. Although coming back to it yeah it is maybe more confusing. I'll just remove the example.